frontend/src/app/login/page.tsx

```diff
@@ -0,0 +1,73 @@
+"use client"
+import React from 'react'
+import { dm_Serif_Display_italic } from '@/app/Fonts/DM_Serif_Display';
+import { Roboto_Font } from '../Fonts/Roboto';
+import GoogleSignupOption from '../Sign-up/GoogleSignupOption';
+import { useState } from 'react';
+import { useRouter } from 'next/navigation';
+import UserAuth from '../ReusableComponents/UserAuth';
+
+export default function page() {
+  const [username, setUsername] = useState("")
+  const [password, setPassword] = useState("")
+  const router = useRouter()
+
+  const handleLogin = async (e:React.FormEvent) => {
+
+    e.preventDefault();
+    const apiForLogin = await fetch("/api/login", {
```

Comment:
the name `apiForLogin` doesn't really accurately describe this object. this is the response from the login API right? so wouldn't it better to call it like `loginResponse`?